lisa/nic.py

```diff
@@ -158,6 +158,14 @@ def get_lower_nics(self) -> List[str]:
     def get_device_slots(self) -> List[str]:
         return [x.pci_slot for x in self.nics.values() if x.pci_slot]
 
+    def get_current_device_driver(self, nic: NicInfo) -> str:
+        cmd = f"readlink /sys/class/net/{nic.upper}/device/driver"
+        found_link = self._node.execute(cmd, expected_exit_code=0).stdout
+        driver_name = found_link.split("/")[-1]
+        assert_that(driver_name).is_not_equal_to("")
```

Comment:
Add description to understand the error, if it happens.

Reply:
The assert_that will raise an error message into the test result. The error message is only information showing in the PowerBI report. The target is to reduce the chance to check the code, and let user know what's wrong by this error message.

Reply:
Ah, sloppy on my part. will fix